### ISSUE TITLE
Audio not working on C630

### ISSUE THREAD

**Issue by mahiuchun** [aarch64-laptops/debian-cdimage#10]
`sudo apt install alsa-ucm-conf=1.2.3-1+linaro3` did not work for me so I installed `1.2.4-2+linaro1` version instead.

```
$ apt policy alsa-ucm-conf
alsa-ucm-conf:
  Installed: 1.2.4-2+linaro1
  Candidate: 1.2.4-2+linaro1
  Version table:
 *** 1.2.4-2+linaro1 500
        500 http://obs.linaro.org/linaro-overlay-sid/sid ./ Packages
        100 /var/lib/dpkg/status
     1.2.4-2 500
        500 http://deb.debian.org/debian sid/main arm64 Packages
```

After installation there is still no audio output. Only "Dummy Output" shows up as an output device.

[dmesg.txt](https://github.com/aarch64-laptops/debian-cdimage/files/5976918/dmesg.txt)

**Comment by shawnguo2** [comment]
Please stay on `1.2.3-1+linaro3` which is the only version we tested, and report how audio doesn't work with some details.

**Comment by mahiuchun** [comment]
`1.2.3-1+linaro3` is not available for me. My APT sources are

```
deb http://deb.debian.org/debian sid main
deb-src http://deb.debian.org/debian sid main
```

and

```
deb http://obs.linaro.org/linaro-overlay-sid/sid/ ./
deb-src http://obs.linaro.org/linaro-overlay-sid/sid/ ./
```

**Comment by shawnguo2** [comment]
Oops, I misunderstood your problem.  I will check it and get back to you soon.

**Comment by shawnguo2** [comment]
We just fixed the problem with a new version `1.2.4-2+linaro2` on `linaro-overlay-sid`.  Please give it a try.
